src/main/ui/src/components/header/searchBar.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
```

Comment:
Refactor the file name into Pascal case like the others please and thank you :)

Reply:
Oh also this component should be in its own folder outside of the header folder.

Reply:
So components should have `Header` `SearchBar` and `Notes` folders all capitalized.